README.md

```diff
@@ -21,6 +21,19 @@ To bind the java agent to a specific IP change the port number to `host:port`.
 
 See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
 
+### Wildfly
```

Comment:
I mean move it to the documentation of the Wildfly project. They've chosen to do things have that break standard java mechanisms, so it's really up to them to deal with the results of that.